docs/contributing/pull-requests/branching.md

```diff
@@ -7,21 +7,26 @@ sidebar_custom_props:
 
 ## Naming Convention
 
-To keep branches organized we adhere to a naming convention for branches. The convention splits
-branch names into 2 components, each separated by a `/`.
-
-1. The developers' pod abbreviation (e.g. `sm`)
-2. The Jira issue tag (e.g. `SM-1234`)
-
-```bash
-# Some examples
-sm/SM-1234
-ps/PS-4321
-```
+To keep branches organized we adhere to a naming convention for branches.
 
 This naming convention allows us to easily identify the type of work being done on a branch and will
 assist in identifying and tracking down stale branches.
 
+### Branches for a specific Jira issue
+
+In order to tie the work on a branch to our Jira instance, the branch name should consist of the
+Jira issue tag (e.g. `PM-1234`). Jira will automatially generate the `git` command with the proper
```

Comment:
This is a personal preference, but I actually find the team prefix to provide more value now that (almost) everyone will use the same jira tag. It's going to be quite difficult tie branches to teams otherwise.

Reply:
I had the same thought, but without Jira's built-in abbreviations, we would need to agree upon them as a team and stay fairly consistent.  I can propose a prefix for each team on the docs if you'd like.  Would you prefer an abbreviation (Auth) or an initialism like Jira uses (AT or SM)?  Or is a combination of both OK?